corehq/apps/cloudcare/management/commands/populate_application_access.py

```diff
@@ -28,4 +28,5 @@ def update_or_create_sql_object(self, doc):
             SQLAppGroup(app_id=group['app_id'], group_id=group['group_id'])
             for group in doc['app_groups']
         ], bulk=False)
+        model.save()
```

Comment:
is there a danger in moving the save action outside of the `atomic` block?

Reply:
The possibility that something else will update the model while the migration is running - pretty unlikely for these web apps permissions docs, but better to be cautious.